samples/bot-all-cards/nodejs/.teamsapptesttool.yml

```diff
@@ -0,0 +1,68 @@
+# yaml-language-server: $schema=https://aka.ms/teams-app-test-tool-config/0.1.0/config.schema.json
+# Visit https://aka.ms/teams-app-test-tool-config-guide for details on this file
+
```

Comment:
.teamsapptesttool.yml file is needed or not?

Reply:
yes this file is needed

Reply:
do we really need these dummy users' information for this sample? if not, we can remove this file and the bot can still work.